services/rfq/relayer/pricer/fee_pricer.go

```diff
@@ -127,14 +137,28 @@ func (f *feePricer) GetDestinationFee(parentCtx context.Context, _, destination
 		metrics.EndSpanWithErr(span, err)
 	}()
 
-	// Calculate the destination fee
-	gasEstimate, err := f.config.GetDestGasEstimate(int(destination))
-	if err != nil {
-		return nil, fmt.Errorf("could not get dest gas estimate: %w", err)
+	fee := big.NewInt(0)
+
+	// Calculate the static L2 fee if it won't be incorporated by directly estimating the relay() call
+	// in addZapFees().
+	if quoteRequest == nil || quoteRequest.Transaction.CallParams == nil {
```

Comment:
Dont we need to check the `CallParams` length for being zero here? This should be present in every V2 quote request, the difference is whether it's empty (no Zap call requested) vs non-empty (providing data for the Zap call).

Reply:
Good call, I'm not sure how the ABI impl handles this but better safe than sorry